dashboard/test/controllers/levels_controller_test.rb

```diff
@@ -3,6 +3,8 @@
 class LevelsControllerTest < ActionController::TestCase
   include Devise::Test::ControllerHelpers
 
+  default_update_blocks_params = nil
```

Comment:
I've followed your suggestion, but FYI that tests were passing for some reason the way that I had it. Not sure I can explain why exactly.